objgraph.py

```diff
@@ -135,7 +135,7 @@ def count(typename, objects=None):
         del objects  # clear cyclic references to frame
 
 
-def typestats(objects=None, shortnames=True):
+def typestats(objects=None, shortnames=True, filter=None):
```

Comment:
This new parameter needs documentation in the docstring, and a `.. versionchanged: ...` directive.

Reply:
Thanks very much, and i have do some fix with your advise, please check again